stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/StreamMessageInputView.kt

```diff
@@ -170,6 +183,12 @@ public class StreamMessageInputView : ConstraintLayout {
     }
 
     private fun configTextInput(typedArray: TypedArray) {
+        suggestionListController = SuggestionListController(binding.suggestionListView, binding.etMessageTextInput)
+
+        binding.etMessageTextInput.doAfterTextChanged {
+            suggestionListController.onTextChanged(it.toString())
```

Comment:
Careful, we're calling the [`Any?.toString()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/to-string.html) extension here, because `it` is an `Editable?`. This will pass in `"null"` if `it == null`.

Using `it?.toString()` instead would return `null` if `it == null`, and we could then default to an empty string with an Elvis operator, or something like that instead.

Reply:
Really nice catch, thanks!

Reply:
Done